From d758e7bce52f4073e55534dc005cc403dc54b6d3 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Mon, 4 Dec 2023 18:19:43 +0100
Subject: [PATCH 0671/1141] dt-bindings: iio: adc: update mdf bindings

Add feature-domains and power-domains properties in STM32 MDF
DT bindings.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: Id7ab4760a95761e6f8c81196f122795885665452
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/343811
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../devicetree/bindings/iio/adc/st,stm32-mdf-adc.yaml     | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Documentation/devicetree/bindings/iio/adc/st,stm32-mdf-adc.yaml b/Documentation/devicetree/bindings/iio/adc/st,stm32-mdf-adc.yaml
index 40cc30e42bf2..49db9e3ffdbd 100644
--- a/Documentation/devicetree/bindings/iio/adc/st,stm32-mdf-adc.yaml
+++ b/Documentation/devicetree/bindings/iio/adc/st,stm32-mdf-adc.yaml
@@ -64,6 +64,14 @@ properties:
     items:
       - const: mdf
 
+  feature-domains:
+    $ref: /schemas/types.yaml#/definitions/phandle-array
+    description: |
+      Phandle to the rifsc device to check access right.
+
+  power-domains:
+    maxItems: 1
+
   st,interleave:
     description: |
       List of phandles of interleaved filters. The indexes of interleaved filters must be
-- 
2.39.2

